vast/vast.cpp

```diff
@@ -220,9 +222,9 @@ int main(int argc, char** argv) {
   if (auto result = run(*invocation, sys, root_factory); !result)
     run_error = std::move(result.error());
   else
-    result->apply({[&](caf::error& err) {
```

Comment:
apply gets removed in CAF 18.6

It constructed caf::message_handler from passed callable and called it with *this. So this is the exact same behaviour as previously

Reply:
That makes sense. I believe there was also a typed message view that you can create from the message, but this is probably safer since it doesn't assert that the typed message view's signature actually matches.